indieauth.php

```diff
@@ -18,9 +18,55 @@
 	define( 'INDIEAUTH_REMOTE_MODE', 0 );
 }
 
+register_activation_hook( __FILE__, array( 'IndieAuth_Plugin', 'schedule' ) );
```

Comment:
I would call it `activate` to match the `deactivate` case

Reply:
It's called on upgrade and when you load the config page as well, to check if the hook is scheduled. Usually, if it was only called on activation, I would call it activate.

Reply:
but then we should also call the `deactivation` hook `deactivation`.

Reply:
Okay.

Reply:
Okay. Done.